src/pages/edge/edge-network/validation/edge-analytics-validation.md

```diff
@@ -0,0 +1,97 @@
+
+## Inspect the Extension Versions View
+
+Add the **Extension Versions** View to the left navigation menu in Assurance. This view allows you to verify which Mobile SDK extensions are being used and their respective version numbers. Ensure that all extensions are up-to-date or at the desired versions.
+If you see a missing extension or the wrong version of an extension, you need to check your Mobile SDK registration implementation in the client app.
+
+![Extension view in the Assurance interface](../assets/validation/extension-view.png)
+
+Depending on the method you are using to send analytics data through the Edge Network, please verify the following extensions:
+
+1. If using Edge.sendEvent:
+
+* Check for: Edge Extension, Edge Identity Extension, Mobile Core Extension
+* Optional: Consent Extension, Lifecycle Extension
+
+2. If using MobileCore.trackAction or MobileCore.trackState:
+
+* Check for: Edge Bridge Extension, Edge Extension, Edge Identity Extension, Mobile Extension
+* Optional: Consent Extension, Lifecycle Extension
+
+Refer to this [documentation](https://developer.adobe.com/client-sdks/solution/adobe-analytics/migrate-to-edge-network/) to learn more about which extension should be used.
+
+## Validate events with the Analytics Event view
+
+The [**Analytics Events**](https://experienceleague.adobe.com/en/docs/experience-platform/assurance/view/adobe-analytics-edge) View offers an in-depth look at SDK events, assisting users in debugging and validating their events. It features a detailed panel that explains how each event was processed, from the client SDK to the Analytics Service.
+
+//TO DO Add screenshot when the new Analytics Event is released.
+
+Let’s walk through an example of how you can use this view to debug an event.
+
+1. **Locate the Event**: Find the event in question within the list of SDK events.
+2. **Open the Details Panel**: Click on the event to open the detailed panel.
+3. **Check Post-Processed Data**: Look at the last event in the chain of events, labeled “POST-PROCESSED DATA.” If the assigned data is present there, it confirms that the data has been correctly passed to Analytics.
+
+![The screenshow shows the context data sends to Post-processed data succcessfully](../assets/validation/edge-bridge-success.png)
+
+4. **Review the chain of events**: If the “POST-PROCESSED DATA” event is missing or the expected data isn’t there, examine the chain of the events to understand how the events were processed. Debug the issue by following the events from the client SDK to the Analytics platform.
+
+![The screenshow shows Edge Bridge Event has wrong data format](../assets/validation/edge-bridge-fail.png)
+
+In this Edge Bridge Example, when checking the Edge Bridge Request event, the contextdata does not match the proper format as specified in the latest Edge Bridge Extension version. We suspect this mismatch could be the main reason causing the data not show to the Post-processed data event.
```

Comment:
Don't love the usage of "We" here. I'd just remove the "We suspect" and go with "This mismatch could be..."